search/search-client/src/main/java/io/camunda/search/clients/PersistentSessionSearchClient.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.clients;
+
+import io.camunda.search.entities.PersistentWebSessionEntity;
+import java.util.List;
+
+public interface PersistentSessionSearchClient {
```

Comment:
Same here regarding naming: `PersistentWebSessionSearchClient`

Reply:
I changed the classname to `PersistentWebSessionClient`.